src/XMakeCommandLine/MSBuild.csproj

```diff
@@ -246,6 +245,10 @@
       <SubType>Designer</SubType>
     </None>
     <None Include="MSBuild.exe.manifest" />
+    <FilesToSign Include="$(OutDir)\$(AssemblyName)$(TargetExt)">
+      <Authenticode>Microsoft</Authenticode>
+      <StrongName>StrongName</StrongName>
+    </FilesToSign>
```

Comment:
This should be left out of the merge because in xplat we did this in a .targets file